cmd/kpod/common.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	is "github.com/containers/image/storage"
+	"github.com/containers/storage"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+func getStore(c *cli.Context) (storage.Store, error) {
+	options := storage.DefaultStoreOptions
+	if c.GlobalIsSet("root") || c.GlobalIsSet("runroot") {
```

Comment:
I removed these two options, but left `--storage-driver` and `--storage-opt`.  I don't know if you wanted those removed as well, but if so, `--storage-driver` should probably stay unless there's a way to define a default storage driver in containers/storage.  Right now it gives an error if there are multiple storage drivers present.